### PR TITLE
feat(f3): display the initial power table CID in the F3 manifest

### PR DESCRIPTION
**Pull request by Stebalien**
## Related Issues

This will make it easier to tell users how to set/verify `F3_INITIAL_POWERTABLE_CID`.

## Proposed Changes

Display the initial power table CID in the `lotus f3 manifest` command output, and make sure it's set if available.

## Checklist

Before you mark the PR ready for review, please make sure that:

- [x] Commits have a clear commit message.
- [x] PR title conforms with [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#pr-title-conventions)
- [x] Update CHANGELOG.md or signal that this change does not need it per [contribution conventions](https://github.com/filecoin-project/lotus/blob/master/CONTRIBUTING.md#changelog-management)
- [x] New features have usage guidelines and / or documentation updates in
  - [x] [Lotus Documentation](https://lotus.filecoin.io)
  - [x] [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [x] Tests exist for new functionality or change in behavior
- [ ] CI is green

### CODE REVIEW

File: chain/lf3/f3.go [hunk 1]
```diff
@@ -184,8 +184,20 @@ func (fff *F3) GetLatestCert(ctx context.Context) (*certs.FinalityCertificate, e
...
+	}
+	cert0, err := fff.inner.GetCert(ctx, 0)
+	if err != nil {
+		return m
+	}
+
+	var mCopy = *m
+	m = &mCopy
+	m.InitialPowerTable = cert0.ECChain.Base().PowerTable
+	return m
```
Comment (masih) [inline chain/lf3/f3.go:200]: Nice

**Comment by Stebalien** [comment]
@masih how have you been testing this? SSH into archioz?

**Review by masih — APPROVED** [review]

**Comment by masih** [comment]
> how have you been testing this? SSH into archioz?

this as in the CLI output?

**Comment by Stebalien** [comment]
> this as in the CLI output?

I'm modifying the `lotus f3 manifest` command.

**Comment by Stebalien** [comment]
Yes, this as in the CLI output.

**Review by Kubuxu — APPROVED** [review]

**Pull request merged**